Documentation/SwiftGenKit Contexts/files.md

```diff
@@ -0,0 +1,60 @@
+# Files parser
+
+## Input
+
+The files parser accepts multiple files or directories, which it will recursively search for files that match the given filter (default `.*`).
+
+The list of files will be converted into a structured directory tree.
+
+## Output
+
+The output context has the following structure:
+
+-`files`:`Array` — list of files in the base directory specified
```

Comment:
I haven't double-checked at how we did format the documentation for the other parsers, but I think it would be cleared here to say that it's an `Array<File>`, then below on line 19 say that "Each `File` has the following properties". wdyt?